openml/datasets/functions.py

```diff
@@ -165,6 +165,36 @@ def _get_cache_directory(dataset: OpenMLDataset) -> str:
     return _create_cache_directory_for_id(DATASETS_CACHE_DIR_NAME, dataset.dataset_id)
 
 
+def list_qualities(verbose=False):
+    """ Return list of data qualities available.
+
+    The function performs an API call to retrieve the entire list of
+    data qualities that are available are computed on the datasets uploaded.
```

Comment:
This sentence appears to be incorrect, I believe removing the 'are available' solves this.